kvrocks.conf

```diff
@@ -902,6 +902,11 @@ rocksdb.write_options.sync no
 # Default: no
 rocksdb.write_options.disable_wal no
 
+# Algorithm to use for WAL compression. none to disable.
+# Supports from rocksdb 9.0. At now only none and zstd are support.
+# Default: none
+rocksdb.wal_compression = none
```

Comment:
We need to add corresponding config item in the code.

Reply:
@aleksraiden Sure.

Reply:
@aleksraiden We can revert kvrocks.conf and then @torwig submit the PR while he gets time.

Reply:
Thanks, done